### PR TITLE
Switch dockerfile.base to alpine

### PR DESCRIPTION
**Pull request by dotslash21**
**Fixes Issue:** #292 
**Old Image Size:** 907MB
**New Image Size:** 780MB (127MB Decrement!)

### CODE REVIEW

File: infra/dockerfile.base [hunk 1]
```diff
@@ -1,13 +1,12 @@
-FROM python:3.7.4-slim
+FROM alpine:latest
```
Comment (marco-c) [inline infra/dockerfile.base]: There is a python alpine image which has Python already installed, I think it's preferable to use that one.

**Comment by dotslash21** [comment]
@marco-c, Yeah It seems that I need to change the `dockerfile.base-nlp` and `dockerfile.commit_retrieval` to work with the new base image.

File: infra/dockerfile.base (hunk 1 above)
Comment (dotslash21) [reply infra/dockerfile.base]: Changed in the latest commit.

**Comment by dotslash21** [comment]
It seems the dockerfiles under http_service also needs to be checked for any bugs due to the switch :(

**Comment by marco-c** [comment]
As discussed in #292, this is not really worth it given the small win in size compared to the Debian-based images.

**Pull request closed without merging**